build.linux32x86/editpharoinstall.sh

```diff
@@ -11,8 +11,11 @@ if [ "$1" = -source ]; then
 else
 	SourceFile="PharoV50"
 fi
-SOURCE=../../sources/$SourceFile.sources
-test -f $SOURCE || SOURCE=../../../sources/$SourceFile.sources
```

Comment:
btw, I'm curious why two folders are being tested for the sources file.  Why not standardise on one?  (In the root folder seems fine.)    

Also, a "sources" folder may confuse newcomers expecting C code. Maybe "imageSources" might be a better folder name, or maybe store them under "image/sources", or store sources in the image folder since SqueakV50.sources is already there ??